modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+
+[id="known-issues-1-16_{context}"]
+== Known issues
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16 and later.
+
+[id="fixed-issues-1-16_{context}"]
+== Fixed issues
+
+* Before this update, when you uninstalled {tekton-hub} by deleting the `TektonHub` CR, the pod of the `hub-db-migration` job was not deleted. With this update, uninstalling {tekton-hub} deletes the pod.
+
+* Before this update, when you used {tekton-results} to store pod logs from pipelines and tasks, the operation to store the logs sometimes failed. The logs would include the `UpdateLog` action failing with the `canceled context` error. With this update, the operation completes correctly. 
+
+* Before this update, when you passed a parameter value to a pipeline or task and this value included more than one variable, and they use both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly; the pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
+
+* Before this update, {tekton-chains} failed to generate correct attestations when a task run included multiple tasks with the same name. For instance, when using a matrix of tasks, the attestation was generated for the first image. With this update, {tekton-chains} generates attestations for all tasks within the task run, ensuring complete coverage.
+
+* Before this update, when you used the `skopeo-copy` task defined in the {pipelines-shortname} installation namespace and set its `VERBOSE` parameter to `false`, the task failed. With this update, the task completes normally.
+
+* Before this update, when using {pac}, if you set the `concurrency_limit` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` or `pipelines-as-code` namespace, which provides default settings for all `Repository` CRs, the {pac} watcher crashed. With this update, the {pac} watcher operates correctly with this setting.
+
+* Before this update, all tasks in {pipelines-shortname} included an extra step compared to the cluster tasks of the same name that were available in previous versions of {pipelines-shortname}. This extra step increased the load on the cluster. With this update, the tasks no longer include the extra step as it is integrated into the first step.
+
+* Before this update, when you used one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the parameter was not interpreted correctly and the task failed. With this update, the task interprets the `CONTEX`T parameter correctly and completes successfully.
```

Comment:
```suggestion
* Before this update, when using one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the parameter was not interpreted correctly, and the task failed. With this update, the task interprets the `CONTEXT` parameter correctly and completes successfully.
```

Reply:
Same issue as the previous comment - "when using... the paramenet was not interpreted correctly" sounds like the same entity was "using" and "not interpreting".

However I do see another issue in this one - this passive voice should be removed. Also the `CONTEXT` typo is very real.

```suggestion
* Before this update, when you used one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the task did not interpret the parameter correctly and the task failed. With this update, the task interprets the `CONTEXT` parameter correctly and completes successfully.
```